src/reducers/ReportsReducer.tsx

```diff
@@ -23,7 +22,8 @@ export const reportsReducer = function (
     action: GenericAction
 ) {
     switch (action.type) {
-        case pendingMessage(FETCH_REPORTS): {
+        // FETCH_REPORTS list of reports
+        case pendingMessage(ActionTypes.FETCH_REPORTS): {
             const nextState: ReportState = {
```

Comment:
I think if you add `ReportState` as a return type on the reducer like `export const reportsReducer = function(...): ReportState`, then you don't need to explicitly create a typed `nextState` object for each of these, you could just `return { ...state, total: 0 }`. Might save you a bunch of lines in these reducers, but again a matter of preference. Feel free to ignore these minor suggestions :)